source/_components/climate.modbus.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: "Modbus"
+description: "Instructions how to integrate a Modbus thermostat within Home Assistant."
+date: 2018-01-29 9:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: modbus.png
+ha_category: Thermostat
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+
+The `modbus` thermostat allows you to use a sensor value (current temperature)
+and target value (target temperature) from [Modbus](http://www.modbus.org/)
+registers.
+
+To use your Modbus thermostat in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+climate:
+  - platform: modbus
+    name: Watlow F4T
+    slave: 1
+    target_temp_register: 2782
+    current_temp_register: 27586
+
+```
+
+{% configuration %}
+  - **name** (*Required*): Name of the sensor.
```

Comment:
Almost correct! You need to change/format the configuration options as well.

https://home-assistant.io/developers/documentation/create_page/#configuration